bsm_helper/include/bsm_helper/bsm_helper.h

```diff
@@ -0,0 +1,29 @@
+#include <string>
```

Comment:
Almost missed this,

You need to add a 
#pragma once 
to the top of this file as an include guard. 
Also all source code files though have the license comment